stock_service_level/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Foodles (http://www.foodles.co).
+# @author Pierre Verkest <pierreverkest84@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock service level",
+    "summary": "Manage stock route by service level",
+    "version": "17.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Warehouse Management",
+    "website": "https://github.com/OCA/wms",
+    "author": "Pierre Verkest <pierreverkest84@gmail.com>,"
```

Comment:
maybe just your name here or it should be Foodles?
```suggestion
    "author": "Pierre Verkest,"
```

Reply:
I'm fine with this suggestion